lib/Raven/Serializer.php

```diff
@@ -67,17 +67,44 @@ public function __construct($mb_detect_order = null)
      * sanitization and encoding.
      *
      * @param mixed $value
+     * @param array $context
+     * @return string|bool|double|int|null|object|array
```

Comment:
I was wrong in saying that the return value of the function should be that because the return types can only be scalars). I would use mixed anyway, because I find it prettier and easier to read, but feel free to do whathever you want on this. Doc is still missing for the `$value` and `$context` arguments and that is my main concern